boden/wege/strasse.h

```diff
@@ -26,7 +26,8 @@ class strasse_t : public weg_t
 	enum {
 		AVOID_CITYROAD   = 0x01, // this street avoid becoming cityroad.
 		CITYCAR_NO_ENTRY = 0x02, // citycar cannot enter this road.
-		USE_GIVEN_HEIGHT = 0x04  // this flag is used only for construction
+		USE_GIVEN_HEIGHT = 0x04, // this flag is used only for construction
+		CAN_BRANCH_CITYROAD = 1U<<4
```

Comment:
getter/setter と表現を揃えた方がよいです

```suggestion
		ALLOW_BRANCH_CITYROAD = 1U<<4
```